crud.py

```diff
@@ -0,0 +1,64 @@
+from fastapi import HTTPException, Response
+from sqlalchemy.orm import Session
+from starlette import status
+from typing import Type
+
+from database import Base
+from models import DBAuthor, DBBook
+from schemas import BookNoId
+
+
+def get_many(db_session: Session, db_model: Type) -> list:
+    return db_session.query(db_model).all()
+
+
+def get_one(ident: int, db_session: Session, db_model: Type) -> Base:
+    return db_session.get(db_model, ident)
```

Comment:
It can return `None` and cause 500 error when FastAPI will try to serialize it before giving response